Add dropbox to eligible HOC prizes

kozzi opened the pull request:
Primary change here was to add dropbox to the HOC prizes page.

Some decisions I made and why:
- I opted to use existing company logos we have on our [partners page](https://hourofcode.com/us/partners) instead of the "fanning gift card" image. This way, we don't need to create new custom assets whenever we add a different kind of gift card. Unless there's a strong preference for the fanning gift card visual?
- The css used to vertically center logo is found here: http://stackoverflow.com/a/18869078/5428277. I tried using `display: table`, `display: table-cell`, `vertical-align: middle` (what the [partners page](https://hourofcode.com/us/partners) does) but that didn't work well when screen is small. If you know a better way, let me know. 
## Before

![image](https://cloud.githubusercontent.com/assets/1885340/11825193/7c6e3fc8-a332-11e5-9d70-21b39db98eb1.png)
## After

![image](https://cloud.githubusercontent.com/assets/1885340/11825210/8e7addca-a332-11e5-84b1-a8d28335e39d.png)

Also tested on mobile.

kozzi commented:
fyi @tanyaparker @breville

tanyaparker commented on pegasus/sites.v3/hourofcode.com/public/css/prizes-page.css:
```diff
@@ -0,0 +1,20 @@
+    left: 0;
+    right: 0;
+    margin: auto;
+}
```
Nits: indentation is different between these two classes. & EOF line

kozzi replied:
Ah yes. Good catch. Too much copying and pasting code from the internet.

Although... since I'm moving the images into a haml view, I'm going to move these pieces of css into common.css and delete this file altogether.

tanyaparker commented on pegasus/sites.v3/hourofcode.com/public/prizes/index.md:
```diff
@@ -17,13 +18,20 @@ Every educator who hosted an Hour of Code for students received 10 GB of Dropbox
-<img styel="float:left;" src="/images/fit-130/microsoft_giftcards.png"/>
+**Every** educator who hosts an Hour of Code is eligible to receive **$10 to Amazon.com, iTunes, Windows Store or 10GB of Dropbox space** as a thank-you gift!*
+
+<div class="prizes-logo">
```
Definitely like using logos over the fan images, but I would include the images as a haml view since it's reused in other parts of the site like hourofcode.com homepage.

tanyaparker commented:
I kinda think apple logo should include iTunes since the others also get a word.

![image](https://cloud.githubusercontent.com/assets/1920530/11826238/f67ca678-a338-11e5-8589-5cdf6a390026.png)

tanyaparker commented:
We would also need to update the view that's on the hourofcode.com homepage, below the gallery of images. Even though it's being taken out later this week-- but will be good to refactor that already for next year.

tanyaparker commented:
I would also check with Kate if there were other parts of the site that mention prizes. But these are the first 2 places that come to mind.

tanyaparker commented:
Caley I can take the "deep dive" look at this one. But feel free to leave your notes too of course.

breville commented:
out of curiosity, what went wrong with the other styling on smaller screens?

kozzi commented:
@breville - It wasn't responsive. When using CSS tables and partners page go <768px, the logos on the right just get cut off instead of wrapping below.

caleybrock commented:
This is much better +1 on the iTunes logo

kozzi commented:
Updates based on feedback:
- Moved giftcards images into its own haml view
- Replaced the giftcards section on homepage with haml view
- Used iTunes logo instead of Apple logo

## HOC homepage

![image](https://cloud.githubusercontent.com/assets/1885340/11850409/f29198f8-a3e1-11e5-99e2-e69a848f3968.png)

## Prizes page

![image](https://cloud.githubusercontent.com/assets/1885340/11850420/041f1348-a3e2-11e5-8c18-3f5fbb663f59.png) 

Note: I deliberately made the sizes of the logos different to make the baselines look _somewhat_ reasonable next to each other (it's still not perfect). It was either this or go in and edit the logo assets themselves to make padding/baselines consistent. The latter felt scarier since they could be used in other places.

breville commented:
good call on not modifying the assets themselves.  Note that pegasus has some special URLs (fit-X, fill-X, etc.) which have the web server modify the image before it serves it back for your particular request.

kozzi commented:
@breville Yup! I used pegasus's url resizing of images for this change.

tanyaparker commented:
lgtm!
